[Feature] - Add capability to specify admission controller certificate from existing secret (#1506)
traefik/traefik-helm-chart: traefik/templates/NOTES.txt

    {{- end -}}
{{- end -}}

{{/* Warn about missing secret when enabling managed certificate with Hub admission controller */}}

> {{- if .Values.hub.apimanagement.admission.selfManagedCertificate }}

Reviewer: ```suggestion
{{- if and .Values.hub.token .Values.hub.apimanagement.enabled .Values.hub.apimanagement.admission.selfManagedCertificate }}
```

  {{- $cert := lookup "v1" "Secret" (include "traefik.namespace" .) $.Values.hub.apimanagement.admission.secretName -}}
  {{- if not $cert }}

>     {{- printf "\nWARNING: %s secret is missing in traefik namespace.\n" $.Values.hub.apimanagement.admission.secretName -}}

Reviewer: ```suggestion
    {{- printf "\nWARNING: webhook secret %s is self managed and I was not able to find it in %s namespace.\n" $.Values.hub.apimanagement.admission.secretName (template "traefik.namespace" . ) -}}
```

>     {{- printf "Please deploy %s secret and restart your Traefik Hub pods.\n" $.Values.hub.apimanagement.admission.secretName -}}

Reviewer: ```suggestion
```
